### ISSUE TITLE
Redirecting to random snp

### ISSUE THREAD

**Issue by danielecook** [NovembreLab/ggv#21]
Just a thought here, and opening this up to hear what others think.

The random snp functionality is definitely cool, but I think if you are searching for a variant having it redirect to a random variant is not particularly helpful. I think a better alternative would be to have a message indicating the variant can not be found and then clear the map of all pie-charts. Thoughts?

**Comment by jnovembre** [comment]
Yes - we could have a go-to "default" prompt that says "Please search on a variant or click the random button."   Make it so!

**Comment by danielecook** [comment]
Will do.
